python/pylibcudf/pylibcudf/null_mask.pyx

```diff
@@ -148,3 +150,28 @@ cpdef tuple bitmask_or(list columns):
         c_result = cpp_null_mask.bitmask_or(c_table.view())
 
     return buffer_to_python(move(c_result.first)), c_result.second
+
+
+cpdef size_type null_count(Py_ssize_t bitmask, size_type start, size_type stop):
```

Comment:
Are you using `Py_ssize_t` over `int` because the `bitmask` is essentially an index to a python buffer object? And according to this [stackoverflow post](https://stackoverflow.com/a/20987501). `Py_ssize_t` preffered over `int` when indexing into python objects.

Reply:
I was matching the `ptr` type of `int_to_void_ptr` here. Additionally, using `int` here would result in an `OverflowError`